prime-router/src/main/kotlin/cli/OktaCommands.kt

```diff
@@ -293,4 +325,48 @@ abstract class OktaCommand(name: String, help: String) : CliktCommand(name = nam
             return Path.of(System.getProperty("user.home"), localPrimeFolder, localTokenFileName)
         }
     }
+}
+
+/**
+ * Container for configurable values regarding Okta interactions.
+ * These values are required for Okta API calls to work.
+ *
+ * @property baseUrl Base endpoint for the Okta account
+ * @property clientId Client ID for the Okta account
+ * @property authKey Api key for the Okta account
+ */
+data class OktaConfig(
+    var baseUrl: String = oktaPreviewBaseUrl,
+    var clientId: String = oktaPreviewClientId,
+    var authKey: String = oktaDummyAccessToken
+) {
+    /**
+     * Load environment variables for the Okta Api
+     * @param oktaApp Configuration values
+     */
+    fun init(oktaApp: OktaCommand.OktaApp) {
+        baseUrl = oktaBaseUrls[oktaApp] ?: error("Invalid app - Okta url")
+        clientId = clientIds[oktaApp] ?: error("Invalid app")
+        authKey = System.getenv("OKTA_authKey")
```

Comment:
This is throwing an NPE for me when I don't have the this env var set.  I'm wondering if we should make this optional?